Tests/ExamplesTest/requirements.txt

```diff
@@ -1,7 +1,7 @@
 attrs==24.2.0
 iniconfig==2.0.0
 py==1.11.0
-pyparsing==3.1.4
+pyparsing==3.2.0
```

Comment:
It seems that Ubuntu 20.04 doesn't support this version. I think we can drop the update directly.